app/handler/view.go

```diff
@@ -47,6 +48,40 @@ func ViewIDGet(db *database.DB) http.HandlerFunc {
 	}
 }
 
+func ViewIDRawGet(db *database.DB) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		reportID, err := strconv.Atoi(chi.URLParam(r, "reportID"))
+		if err != nil {
+			template.ErrorTemplate(w, r, "Please specify a report", http.StatusNotFound)
+			return
+		}
+
+		var reporterName string
+		err = db.Get(&reporterName, "SELECT reporterName FROM crash_reports WHERE id = ?", reportID)
+		if err != nil {
+			log.Printf("can't find report %d in database: %v", reportID, err)
+			template.ErrorTemplate(w, r, "Report not found", http.StatusNotFound)
+			return
+		}
+
+		report, err := db.FetchReport(int64(reportID))
```

Comment:
It would be better to fetch the raw report here. Some fields get lost in the marshal/unmarshal because they aren't used.